arrowFn.js

```diff
@@ -0,0 +1,28 @@
+//1
+let toggle=false;
+
+const func=()=>{
+   toggle=true;
+}
+
+func();
+
+console.log(toggle===false ?"Toggle False":"Toggle True")
+
+
+
+//2
+let n=10;
+
+let obj1={
+    a:10,
+    b:()=>{console.log(this.n); //undefined as arrow doesn't possess power of this
+     console.log(this.a); //undefined as arrow doesn't possess power of this 
+    },
+    c:function(){console.log(this.n); //undefined as variable ain't defined within scope of the object obj1
```

Comment:
do not write comments that explain how we are getting a certain value in output.  

Comment should only be used for explaining `why a certain thing is done` . 
Comments should not explain `how a certain thing is done`. It is assumed that everybody understands the `how part`.